croncert-config.yml

```diff
@@ -699,6 +699,59 @@ scrapers:
           selector: div.row.sk_indent.gig.xs-card > div.col-xs-12.col-sm-8.col-md-7.card > div.hidden-xs > div.tickets > a.boxoffice.ticket_link.information
           attr: href
 
+  - name: DClub
+    url: https://dclub.ch/agenda/
+    item: div.dclub-agenda > div.date_container > div.events_container > div.event
+    fields:
+      - name: "type"
+        value: "concert"
+      - name: "city"
+        value: "Lausanne"
+      - name: "location"
+        value: "D! Club"
+      - name: "sourceUrl"
+        value: "https://dclub.ch/agenda/"
+      - name: style
+        type: text
+        location:
+          selector: div.info > div.details > p.style
+      - name: url
+        type: url
+        location:
+          selector: a.main
+          attr: href
+      - name: title
```

Comment:
For the title we have the same problem that we discussed in the Dachstock issue. I opened an issue in goskyr for that: https://github.com/jakopako/goskyr/issues/188

Reply:
Nevertheless I'm merging this config. We can update it once the new functionality is ready.